packages/ui/src/message-room-details-dialog/MessageRoomDetailsDialog.jsx

```diff
@@ -59,6 +59,42 @@ function MessageRoomDetailsDialog({
     setShowLeaveChannelModal(!showLeaveChannelModal);
   const toggleArchiveChannel = () => setShowArchiveChannel(!showArchiveChannel);
 
+  const [description, setDescription] = useState("");
+  const [roomData, setRoomData] = useState(null);
+
+  useEffect(() => {
+    let isFetched = true;
+
+    const path = window.location.pathname;
+    const newPath = path.split("/");
+    const room_id = newPath[4];
+
+    const organization = JSON.parse(localStorage.getItem("userData"));
+    const org_id = organization.user.org_id;
+
+    const fetchData = async () => {
+      const res = await fetch(
+        `https://chat.zuri.chat/api/v1/org/${org_id}/rooms/${room_id}`
+      );
+
+      if (!res.ok) {
+        throw Error("Could not fetch data");
```

Comment:
show the error properly

Reply:
us a notification

Reply:
What do you mean by "as a notification" chief?

Reply:
a toast alert
a small colored box on the screen that shows you the alert instead of throwing an error